client/signup/steps/design-type-with-store/index.jsx

```diff
@@ -36,6 +39,48 @@ export default React.createClass( {
 		this._pressableStore = pressableStoreStep;
```

Comment:
Any particular reason this has a leading underscore? We don't usually use that style here. Is there anything wrong with using `this.pressableStore` instead?